dummy-data/actual-items.js

```diff
@@ -0,0 +1,31 @@
+const actualItems = [
```

Comment:
Please move the mock data files from graphql to the MSW folder in wallora.com repo